src/index.ts

```diff
@@ -33,6 +33,10 @@ interface GlobalConfig {
    * @default false
    */
   forceWrap: boolean;
+  /**
+   * Custom logger.
+   */
+  logger?: any;
```

Comment:
Would you be able to create an interface for this?

Reply:
Done